pages/public_cloud/public_cloud_databases/mongodb_15_monitoring/guide.en-gb.md

```diff
@@ -0,0 +1,132 @@
+---
+title: MongoDB - Monitoring a MongoDB cluster
+excerpt: Learn how to monitor your databases via the OVHcloud Control Panel, using a CLI or a GUI
+updated: 2024-01-31
+---
+
+## Objective
+
+Monitoring MongoDB databases is crucial for detecting glitches and slowdowns, which in turn allows you to make sure a system is working properly, particularly in terms of availability, reliability, and scalability.
+
+You can monitor your databases via the OVHcloud Control Panel (and also view summary and history), using a command line interface (CLI), or a graphical user interface (GUI).
+
+**In this guide, we will show you three ways to get and use best practice tips on a daily basis to keep your MongoDB databases in top shape.**
+
+## Requirements
+
+- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account.
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
+- An OVHcloud managed MongoDB service. If you do not have one, please refer to the following guide: [Getting started with Public Cloud Databases](/pages/public_cloud/public_cloud_databases/databases_01_order_control_panel).
+- An account you can use to log in to MongoDB. If you do not have one, please refer to the following guide: [Configure your MongoDB instance to accept incoming connections](/pages/public_cloud/public_cloud_databases/mongodb_02_manage_control_panel).
+
+## Instructions
+
+### Using the metrics provided by OVHcloud
+
+You can use the main metrics and statistics in the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) to track and manage your database instance.
+
+For each Public Cloud Database service, in the `Metrics`{.action} tab, you can find the following charts:
+
+- CPU used (%)
+- Disk space used
+- Number of read IOPS
+- Number of write IOPS
+- Memory used
```

Comment:
No "Memory used" for MongoDB Discovery, only for higher range (Production, advanced)

Reply:
Thank you, I've updated this line to document it